network/monuments.csv

```diff
@@ -113,6 +113,7 @@ NPLY,50227M001,Forced Centering,Pillar,-1.53,Reinforced Concrete,2,2003-03-20T00
 NRRD,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.516,Stainless Steel Rods,10,2008-03-27T00:00:00Z,9999-01-01T00:00:00Z,Greywacke,Greywacke
 NRSW,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.585,Stainless Steel Rods,10,2011-01-18T00:00:00Z,9999-01-01T00:00:00Z,,
 NSFT,,Forced Centering,Steel Mast,-3.65,Reinforced Concrete,0,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z,,
+NTGT,,Forced Centering,Steel Mast,-2.59,Reinforced Concrete,1,2024-01-15T00:00:00Z,9999-01-01T00:00:00Z,,
```

Comment:
Hey @EvanCGNS ! I think it is better to use `0` values for `Foundation depth` as we did for `NSFT` for example.